django_freeradius/base/models.py

```diff
@@ -398,12 +398,30 @@ class AbstractRadiusAccounting(models.Model):
                                        blank=True)
     framed_ip_address = models.GenericIPAddressField(verbose_name=_('framed IP address'),
                                                      db_column='framedipaddress',
-                                                     db_index=True,
                                                      # the default MySQL freeradius schema defines
                                                      # this as NOT NULL but defaulting to empty string
                                                      # but that wouldn't work on PostgreSQL
                                                      null=True,
                                                      blank=True)
+    framed_ipv6_address = models.GenericIPAddressField(verbose_name=_('framed IPv6 address'),
+                                                       db_column='framedipv6address',
+                                                       null=True,
+                                                       blank=True)
+    framed_ipv6_prefix = models.CharField(verbose_name=_('framed IPv6 prefix'),
+                                          max_length=44,
+                                          db_column='framedipv6prefix',
+                                          null=True,
+                                          blank=True)
+    framed_interface_id = models.CharField(verbose_name=_('framed interface ID'),
+                                           max_length=19,
+                                           db_column='framedinterfaceid',
+                                           null=True,
+                                           blank=True)
+    delegated_ipv6_prefix = models.CharField(verbose_name=_('delegated IPv6 prefix'),
+                                             max_length=44,
+                                             db_column='delegatedipv6prefix',
+                                             null=True,
+                                             blank=True)
```

Comment:
can you please write a validator (eg: `ipv6_network_validator`) which ensures that the value, if not falsy, is an ipv6 subnet?

See django validators: 

- https://docs.djangoproject.com/en/3.0/ref/validators/

Use the ipaddress library (standard python 3 library):

- create an ip_network object: https://docs.python.org/3/library/ipaddress.html#ipaddress.ip_network
- if it fails with an exception, raise a validation error (eg: `raise ValidationError('Invalid ipv6 prefix: {}'.format(e))`)
- if the resulting object is not an instance of `IPv6Network` raise a validation error, eg: `raise ValidationError('{} is not an IPv6 prefix'.format(value))`

Write a test for these cases.